webdriver-spec.html

```diff
@@ -1244,6 +1244,12 @@
   time that commands will block before returning with a <a>timeout</a>
   error.
 
+  <p class=note>Navigation actions - Get, Back, Forward, Refresh - always,
+  unless they abort the navigation, complete with the <a>current top-level browsing context</a>
+  being the <a>current browsing context</a>. Even if we navigate Back or Forward to a state
+  where a different <a href=https://html.spec.whatwg.org/#browsing-context>browsing context</a>
+  was the <a>current browsing context</a>.
```

Comment:
Correction to my last comment:

s/Back and Forward commands can complete returning success in certain cases, specifically/Back and Forward commands can complete returning success in certain cases **where the navigation wasn't actually performed**, specifically/

Putting what I wanted to say in the last comment in other words, I would have used a simple language like [1] if a successful response from those actions meant that the navigation has been successfully performed, but that is not guaranteed to always be the case. For that reason I use more verbose and complex language to stay logically correct.

Hope this makes some sense.

[1] <a>current browsing context</a> is guaranteed to be the <a>current top-level browsing context</a> after a successful response sent by a navigation command (Get, Back, Forward or Refresh) **even** if it wasn't before.